Sequences/MRML/vtkMRMLSequenceNode.cxx

```diff
@@ -646,24 +646,22 @@ std::string vtkMRMLSequenceNode::GetDefaultStorageNodeClassName(const char* file
     return "";
   }
 
-  // Use specific volume sequence storage node, if possible
-  std::vector< vtkSmartPointer<vtkMRMLStorageNode> > specializedStorageNodes;
-  specializedStorageNodes.push_back(vtkSmartPointer<vtkMRMLVolumeSequenceStorageNode>::New());
-  specializedStorageNodes.push_back(vtkSmartPointer<vtkMRMLLinearTransformSequenceStorageNode>::New());
-  for (std::vector< vtkSmartPointer<vtkMRMLStorageNode> >::iterator specializedStorageNodeIt = specializedStorageNodes.begin();
-    specializedStorageNodeIt != specializedStorageNodes.end(); ++specializedStorageNodeIt)
-  {
-    if (filename && (*specializedStorageNodeIt)->GetSupportedFileExtension(filename, false, true).empty())
-    {
-      // cannot write into the required file extension
-      continue;
-    }
-    if ((*specializedStorageNodeIt)->CanWriteFromReferenceNode(this))
+  // Use specific sequence storage node, if possible
```

Comment:
It is strange that vtkMRMLSequenceNode::CreateDefaultStorageNode() does not use GetDefaultStorageNodeClassName. Could we completely remove vtkMRMLSequenceNode::CreateDefaultStorageNode() method?

Reply:
Yes, I think that it could be removed.

Reply:
Apparently the method is still needed. Default implementation returns NULL:
https://github.com/Slicer/Slicer/blob/4d4b63d4a0a00f8af0ca9ffa82140e17387729a2/Libs/MRML/Core/vtkMRMLStorableNode.cxx#L374-L378